### ISSUE TITLE
Branch cleanup

### ISSUE THREAD

**Issue by wipfli** [maplibre/maplibre-navigation-ios#84]
Thanks for all the good contributions to this repo! We have seen quite some activity in the last few months which is nice.

Regarding branches, I suggest someone with knowledge of what is what goes through them and deletes all unused branches.

Usually in other repos in the MapLibre org we do the following:

- normal contributions are made on branches of a fork, for example on the repo `wipfli/maplibre-navigation-ios`
- larger contributions that take weeks to months and that are composed of many individual pull requests can be made on branches in the maplibre repo, i.e., `maplibre/maplibre-navigation-ios`

Would this approach also make sense to the contributors/maintainers of maplibre-navigation-ios?

**Comment by hactar** [comment]
I think the "delete branch after merge" setting may be off? I think most of the branches are probably stale after they got merged into main. I suggest turning that feature on to prevent branches from accumulating.

**Comment by wipfli** [comment]
Do you mean this one @hactar ?

![image](https://github.com/user-attachments/assets/367cde59-6588-492d-8bfc-e871fa218858)

**Comment by hactar** [comment]
Yes. We still need to clean up the existing branches ourselves, but at least this would prevent the growth of unused branches in future.

**Comment by wipfli** [comment]
OK I changed that setting.

**Comment by hactar** [comment]
I've browsed through them, cleaned up my own, and deleted a few that were clearly debug branches or commitless. I've also noticed that nearly all stale branches are by @birkskyum - maybe @birkskyum can tell us if there's any hidden gems in the branches we have?

**Comment by birkskyum** [comment]
Oh really, I'll go through them later today

**Comment by birkskyum** [comment]
I honestly don't know why I'm associated to all these branches. None of them appear familiar to me.
